app/views/map.template.njk

```diff
@@ -72,8 +70,13 @@
   const parcels = {{ parcels | dump | safe }}
   const center = {{ center | dump | safe }}
   const sbi = {{ sbi }}
+  const apiKey = {{ apiKey | dump | safe }}
+  const mapStyle = {{ mapStyle | dump | safe }}
 
-  map.displayMap(sbi, parcels, center)
+  if (mapStyle !== '') { 
+    document.getElementById("layer-select").value = mapStyle
+  }
 
+  map.displayMap(apiKey, sbi, parcels, center)
```

Comment:
Could we wrap this in an if statement so it doesn't show the map if there is no API key?